docs/core/project-sdk/msbuild-props.md

```diff
@@ -404,6 +404,9 @@ The `CodeAnalysisTreatWarningsAsErrors` property lets you configure whether code
 </PropertyGroup>
 ```
 
+> [!NOTE]
+> This property applies specifically to the built-in analyzers in the .NET 5+ SDK. It should not be used when you install a NuGet code analysis package.
```

Comment:
When I have the NuGet package and also not specifying EnableNETAnalyzers, there is still a warning if I'm using an older package than what comes with the SDK:

>  warning : The .NET SDK has newer analyzers with version '6.0.0' than what version '5.0.3' of 'Microsoft.CodeAnalysis.NetAnalyzers' package provides. Update or remove this package reference.

Is there a way to pin to a specific version without getting warnings? (which usually should fail CI)

Reply:
@mavasani Should we advertise the `<_SkipUpgradeNetAnalyzersNuGetWarning>` property? It seems to defeat the purpose of the design of generating a warning when the SDK contains newer analyzers. If people want to decouple analyzer updates from SDK updates, then should we really generate a warning, or should it just be a suggestion?

Reply:
I created a new issue to track that.